src/main/java/synapseawsconsolelogin/Auth.java

```diff
@@ -148,8 +148,12 @@ private String getClientIdSynapse() {
 		return result;
 	}
 	
-	private String getClientSecretSynapse() {
-		String result =  getProperty("SYNAPSE_OAUTH_CLIENT_SECRET");
+	String getClientSecretSynapse() {
+		String clientSecretParameterName = getProperty("SYNAPSE_OAUTH_CLIENT_SECRET_PARAMETER_NAME", false);
```

Comment:
This seems like a one off?  It would be nice to store all parameters in the SSM or add more secrets to the SSM without much fuss.   It would be better to make this generic to provide flexibility and make it more maintainable.

Reply:
It's not clear what you are aiming for here:  Previously you asked for a series of changes to support multiple (dev vs. prod) properties files.  Now it sounds like you want to move away from those files and instead store everything (properties as well as credentials) in SSM.    Can we move away from this iterative approach and instead have some specific requirements?  I think that would get us to the finish line quicker.